kahuna/public/js/services/image-logic.js

```diff
@@ -61,12 +61,25 @@ imageLogic.factory('imageLogic', ['imageAccessor', function(imageAccessor) {
         });
     }
 
+    function isSyndicated(image) {
+        const syndicationRights = image.data.syndicationRights;
+
+        const hasSyndicationRights = !!syndicationRights;
+
+        if (!hasSyndicationRights) {
+            return false;
+        }
+
+        return syndicationRights.rights.find(_ => _.acquired === true);
```

Comment:
`rights` is a list, `isSyndicated` is true if at least one of the rights has been acquired